### PR TITLE
create user, login-logout

### PR DESCRIPTION
**Pull request by vuducluan**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -11,6 +11,8 @@ gem "jbuilder", "~> 2.0"
 gem "sdoc", "~> 0.4.0", group: :doc
 gem "bootstrap-sass"
 gem "devise"
+gem "factory_girl_rails"
```
Comment (hellovietnam93) [inline Gemfile]: put in block group development

File: Gemfile [hunk 2]
```diff
@@ -11,6 +11,8 @@ gem "jbuilder", "~> 2.0"
...
+gem "faker"
```
Comment (hellovietnam93) [inline Gemfile]: the same as above

File: app/views/devise/sessions/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<h1><%= t "sign_in" %></h1>
+<div class="col-md-6 col-md-offset-3">
+<%= form_for resource, as: resource_name, url: session_path(resource_name) do |f| %>
```
Comment (hellovietnam93) [inline app/views/devise/sessions/new.html.erb]: ident

File: app/views/devise/registrations/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+      <%= f.text_field :chatworkid, class: "form-control" %>
+    </div>
+    <div class="form-group">
+      <%= f.label :email %><br />
+      <%= f.email_field :email, autofocus: true, class: "form-control" %>
+    </div>
+    <div class="form-group">
+      <%= f.label :password %>
+      <% if @minimum_password_length %>
+        <em>(<%= @minimum_password_length %> <%= t("characters_minimum") %>)</em>
```
Comment (lyxuansang91) [inline app/views/devise/registrations/new.html.erb]: bỏ ngoặc chỗ i18n đi nhé

Comment (hellovietnam93) [inline app/views/devise/registrations/new.html.erb]: <%= t "characters_minimum" %>

**Comment by hellovietnam93** [comment]
OK

File: app/assets/stylesheets/custom.css.scss [hunk 5]
```diff
@@ -103,3 +103,8 @@ $navbar-inverse-border: #e7e7e7;
   background-color: $navbar-inverse-bg;
   border-color: $navbar-inverse-border;
 }
+
+.navbar-inverse .navbar-nav > .open > a:hover, .navbar-inverse .navbar-nav > .open > a:focus {
+    background-color: #e7e7e7;
```
Comment (dieunb) [inline app/assets/stylesheets/custom.css.scss]: indent

**Pull request merged**